File: COOLSOUND-BACKEND/coolsound/src/main/java/com/java/coolsound/service/ComentarioService.java
package com.java.coolsound.service;

import java.util.List;

import com.java.coolsound.model.Comentario;

public interface ComentarioService {

	/**
	 * Gets the all Comentarios.
	 *
	 * @return the all Comentarios
	 */
	List<Comentario> getAllComentarios();
	
	/**
	 * Eliminar Comentario.
	 *
	 * @return eliminar comentario
	 */
	int eliminarComentario(int idComentario);
	
	/**
	 * Añadir Comentario.
	 *
	 * @return añadir Comentario
	 */
	int nuevoComentario(int idHilo, String texto, int idUsuario);
}

File: COOLSOUND-FRONTEND/coolsound/src/app/services/comentario.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { Observable } from 'rxjs';
import { Comentario } from '../model/comentario';

@Injectable({
  providedIn: 'root'
})

export class ComentarioService {

  private baseUrl = '/coolsound/comentarios/';

  constructor(private http: HttpClient) { }

  /** Método para sacar las canciones de BD */
  comentario(): Observable<any> {
    return this.http.get<Comentario>(`${this.baseUrl}` + 'getComentarios/');
  }

  /** Método para eliminar Comentarios de BD */
  eliminarComentario(idComentario: number): Observable<any> {
    return this.http.get(`${this.baseUrl}` + 'eliminarComentario/' + idComentario);
  }

  /** Método para añadir Comentarios a BD */
  nuevoHilo(idHilo: number, texto: string, idUsuario: number): Observable<any> {
    return this.http.get(`${this.baseUrl}` + 'nuevoComentario/' + idHilo + '/' + texto + '/' + idUsuario);
  }
}
